debian/control

```diff
@@ -29,36 +29,36 @@ Architecture: any
 Depends:
     apt-transport-https,
     dconf-cli,
-    dmsetup,
     distro-info-data,
+    dmsetup,
+    e2fsprogs,
     gawk,
     graphicsmagick,
-    init-system-helpers,
     initscripts | sysvinit-utils,
+    init-system-helpers,
     librsvg2-bin,
     locales,
     ncurses-term,
-    psmisc,
-    procps,
-    util-linux,
-    e2fsprogs,
     parted,
-    python3-qubesdb,
+    procps,
+    psmisc,
+    python3-dbus,
     python3-gi,
+    python3-qubesdb,
     python3-xdg,
-    python3-dbus,
-    qubes-utils (>= 3.1.3),
     qubes-core-qrexec (>= 4.2.19),
     qubesdb-vm,
+    qubes-utils (>= 3.1.3),
```

Comment:
That's interesting ordering. I think I see how this happened (basically, sorted with ignoring dashes), but is it actually desirable? There are few other places here where ignoring dashes affect sort order (for example `init-system-helpers`)